packs/feat-effects/effect-strained-metabolism.json

```diff
@@ -24,7 +24,41 @@
         "level": {
             "value": 1
         },
-        "rules": [],
+        "rules": [
+            {
+                "inMemoryOnly": true,
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Item.Enfeebled"
+            },
+            {
+                "inMemoryOnly": true,
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Item.Clumsy"
+            },
+            {
+                "inMemoryOnly": true,
+                "predicate": [
+                    "parent:badge:value:2"
+                ],
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Item.Fatigued"
+            },
+            {
+                "inMemoryOnly": true,
+                "predicate": [
+                    "parent:badge:value:3"
+                ],
+                "alterations": [
+                    {
+                        "mode": "override",
+                        "property": "badge-value",
+                        "value": 2
+                    }
+                ],
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Item.Sickened"
+            }
```

Comment:
When you use `inMemoryOnly` it locks the effect value at that state, with no way to remove it until the granting effect is removed. In the case of the sickened here or the frightened on the other one those can be removed as normal. The frightened is fine to just leave unpredicated and without the in memory grant since that fires off when you get the mild backlash every time, but the sickened here won't be doable properly.

Reply:
But without ```inMemoryOnly``` sickened will not work correctly, when you apply effect with 3 stage (Severe) all is ok, sickened is added, but if you apply effect with 1 stage (Mild) and after it update to 3 condition not added

Reply:
https://github.com/foundryvtt/pf2e/assets/6688366/b8162aff-5c67-4d59-b161-a4349a7b5081